modules/social_features/social_event/modules/social_event_an_enroll/src/Form/EventAnEnrollActionForm.php

```diff
@@ -92,7 +92,7 @@ public function buildForm(array $form, FormStateInterface $form_state, Node $nod
           ],
           'data-dialog-type' => 'modal',
           'data-dialog-options' => json_encode([
-            'title' => t('Enroll in @event Event', ['@event' => $node->getTitle()]),
+            'title' => t('Enroll in') . ' ' . strip_tags($node->getTitle()),
```

Comment:
the word Event has been removed from the title now, that is why the behat test is failing for PAC f.e. 
should we put that back?

Reply:
AH, that explains. I prefer to remove the word "Event" as it doesn't add value here and only makes the title a lot bigger. I'll change the test in PAC.

Reply:
Test is passing now :-)